### PR TITLE
comment changes only to test travis build which seems to be failing

### PR DESCRIPTION
**Pull request by ozydingo**

### CODE REVIEW

**Comment by ozydingo** [comment]
So it looks like Travis CI builds for all PRs are failing (this PR as you can see contains no meaningful changes from master). I don't know where to look from here but am happy to contribute time to help figure it out. Any pointers to fix the CI build process?

**Comment by rosskevin** [comment]
`Gem::InstallError: rack requires Ruby version >= 2.2.2.`

It looks like the travis.yml needs to restrict rubies to 2.2.2 and above.

File: .travis.yml [hunk 1]
```diff
@@ -6,8 +6,6 @@ script: "bundle exec rake"
 
 services: mongodb
 rvm:
-  - 2.1
-  - 2.0.0
   - 2.2
```
Comment (rosskevin) [inline .travis.yml]: needs to be `2.2.2` not `2.2`

**Comment by rosskevin** [comment]
Sorry, I don't get notified when additional commits are made.

**Pull request merged**